index.html

```diff
@@ -16,9 +16,8 @@
 <body>
 	<div class="top-section">
 		<h1>Tic - Tac - Toe</h1>
-		<h2>Current Turn: <span id="currentplayer">Player 1</span></h2>
-		<h2 class="display-results"></h2>
-		<button class="secretbutton">Restart Game?</button>
+		<h2><span id="status">Current Turn: </span><span id="currentplayer">Player 1</span></h2>
+		<button id="restart" style="visibility: hidden;">Restart Game?</button>
```

Comment:
I used `visibility: hidden` here instead of `display: none` is this also made the ui jumpy. `visibility: hidden` hides the content, where `display: none` hides the element.